o_el_iks.API/Entities/AuctionData.cs

```diff
@@ -2,9 +2,41 @@
 
 public class AuctionData
 {
-    public float price { get; set; }
-    public string location { get; set; } 
-    public DateTimeOffset dateOfStart { get; set; }
-    public DateTimeOffset dateOfEnd { get; set; }
-    public TypeOfItem condition { get; set; }
-}
\ No newline at end of file
+    private float _price;
+    public float Price
+    {
+        get => _price;
+        set
```

Comment:
nie no tak getów setów na pewno nie chcemy mieć, jeśli już to zrób jedną metodę która zmienia wartości wszystkich propetek które jej podamy i wtedy tam też zawrzemy informację o dacie ostatniej modyfikacji